tests/integ/test_dataframe.py

```diff
@@ -2085,6 +2085,43 @@ def test_describe(session):
         ],
     )
 
+    mixed_identifiers_dataframe = session.create_dataframe(
+        data=[
+            [1, Decimal("1.0"), "a", 1, 1, "aa"],
+            [2, Decimal("2.0"), "b", None, 2, "bb"],
+        ],
+        schema=["ほげ", "ふが", "a_ほげ", "ふが_1", "a", "b"],
+    )
+
+    assert mixed_identifiers_dataframe.describe().columns == [
```

Comment:
Could we also try several other methods with column names containing special chars? I think it shouldn't only apply to describe.